stable/artifactory-ha/README.md

```diff
@@ -1244,6 +1244,18 @@ Then install nginx-ingress with the values file you created:
 helm upgrade --install nginx-ingress --namespace nginx-ingress center/kubernetes-ingress-nginx/ingress-nginx -f values.yaml
 ```
 
+### Prometheus Metrics
+
+If you want to enable Prometheus metrics you can use the `metrics` and `fluentd` configuration options.
```

Comment:
Is fluentd required for Prometheus metrics? I'm not sure the coupling in the docs is justified. Maybe it should be a different section for each of them. WDYT?

Reply:
Yeah that's a good point, I'll break them out.